src/assets/scss/_record.scss

```diff
@@ -43,6 +43,11 @@
               font-size: 1rem;
               padding-top: 7px;
               width: 330px;
+
+              // overriding the title-button class
+              & > * {
+                display: inline-flex;
+              }
             }
```

Comment:
added this to override title-button class, giving the change there was causing the export and permlink button to come in two lines